setup.ts

```diff
@@ -45,6 +47,28 @@ async function accessAndRefreshTokens(
   }
 }
 
+function transactionLogPath(logPath: string | null): void {
+  const config = dotenv.parse(fs.readFileSync(".env"));
+  config.TRANSACTION_LOG_PATH =
+    logPath ?? path.resolve(__dirname, "logs/transaction.log");
```

Comment:
If it's an absolute path, then `logs` must be removed. You are adding to the path without verifying or creating the `logs` directory.

Reply:
Done